Vagrantfile

```diff
@@ -58,10 +58,19 @@ sudo apt-get install rvm -y
 
 sudo usermod -a -G rvm $USER
 
+# Setup mastodon user & home directory for using systemd:
+sudo adduser --disabled-login --gecos "" mastodon
+sudo usermod -a -G rvm mastodon
+sudo ln -nsf /vagrant/ /home/mastodon/live
```

Comment:
Whilst this isn't _strictly_ necessary, it is very helpful for development to be able to test out the systemd unit files inside the vagrant machine.